test-menus.py

```diff
@@ -0,0 +1,94 @@
+import unittest
+from mock import patch
+from decorators import get_menu_entries
+
+class MenuTestCase(unittest.TestCase):
```

Comment:
These tests are good for the `get_menu_entries` method in general, but you should only test a single function with the tests. Now you test both `get_menu_entries` and `get_permissible_entries`. Add some separate tests for the latter and mock that function while testing the first.

Reply:
Working on it :+1: